src/API/src/db/occurrence/occurrence.service.ts

```diff
@@ -27,6 +29,7 @@ export class OccurrenceService {
       .orderBy('occurrence.id')
       .leftJoinAndSelect('occurrence.sample', 'sample')
       .leftJoinAndSelect('occurrence.site', 'site')
+      .leftJoinAndSelect('occurrence.recordedSpecies', 'recordedSpecies')
```

Comment:
Does this actually get the species information out, or just the recordedSpecies?

Reply:
I think it gets the species info out as it is available in the playground as a child of recordedSpecies, and then it's that species information which is used to colour the markers